source/Octopus.Tentacle/Commands/CreateInstanceCommand.cs

```diff
@@ -1,15 +1,15 @@
 using System;
-using Octopus.Shared.Configuration.Instances;
-using Octopus.Shared.Startup;
+using Octopus.Tentacle.Configuration.Instances;
+using Octopus.Tentacle.Startup;
 
 namespace Octopus.Tentacle.Commands
 {
     public class CreateInstanceCommand : AbstractCommand
     {
         readonly IApplicationInstanceManager instanceManager;
-        string instanceName;
-        string config;
-        string home;
+        string instanceName = null!;
```

Comment:
Do we want to revisit these `null!` at some point to clean them up? Think they're fine to keep in as part of this PR.

Reply:
Yes, I might change these back for the test project so they are easier to spot if we flip nullable on.

Reply:
There is some "prior art" in the tests that uses nullable and `null!` initializers. Will leave them as-is for now, tech-debt to be paid off in the future.